elasticizer/__init__.py

```diff
@@ -1 +1,2 @@
 from elasticizer.pipelines import *
+from elasticizer import __main__
```

Comment:
@JeffreyMFarley I added this so I could test main, but maybe there's another way to do that?